plugins/generator-1.16.5/forge-1.16.5/templates/item.java.ftl

```diff
@@ -110,10 +110,29 @@ import net.minecraft.entity.ai.attributes.Attributes;
 					.isImmuneToFire()
 					</#if>
 					.rarity(Rarity.${data.rarity})
+					<#if data.isFood>
+					    .food((new Food.Builder())
+					            .hunger(${data.nutritionalValue})
+					            .saturation(${data.saturation}f)
+                                <#if data.isAlwaysEdible>.setAlwaysEdible()</#if>
+                                <#if data.isMeat>.meat()</#if>.build())
+                    </#if>
 			);
 			setRegistryName("${registryname}");
 		}
 
+        <#if data.animation != "eat">

```

Comment:
This will set getUseAction for all cases except for eat. In case of default animation for food or non food item, this method should not be generated

Reply:
The default case is `"none"` IIRC?

Reply:
No, none will set no animation at all. If you check the current food template, it checks for the eat animation.

Reply:
Yes, but now this is normal item, so you need to handle cases for items too, not just food